Functions/Definition/task.md

```diff
@@ -1,11 +1,14 @@
 ## Definition
 
-Functions are a convenient way to divide your code into useful blocks, make it more 
+Functions are a convenient way to divide your code into useful blocks, make it more
 readable, and reuse it.
-The keyword `def` introduces a function definition. 
-It must be followed by the function name and the parenthesized list of formal parameters (which can be empty). 
+The keyword `def` introduces a function definition.
+It must be followed by the function name and the parenthesized list of **formal parameters** (which can be empty).
 The statements that form the body of the function start at the next line and must be indented.
 
+<details>
+Formal parameters are enclosed in parentheses; they are the variables defined by the function that receive values when the function is called. The list consists of variable names of all the necessary values for the method. Each formal parameter is separated by a comma. When method is not accepting any input values, then the method should have an empty set of parentheses after the method name. e.g. <code>addition()</code>.
```

Comment:
Formal parameters are enclosed in parentheses; they are the variables defined by the function, which receive values when the function is called. The list consists of variable names of all the necessary values for the method. Each formal parameter is separated by a comma. When the method is not accepting any input values, it should have an empty set of parentheses after the method name, e.g., <code>addition()</code>.